src/main/java/eu/europa/ted/eforms/sdk/analysis/fact/NoticeTypesIndexFact.java

```diff
@@ -17,6 +19,20 @@ public NoticeTypesIndexFact(NoticeTypesForIndex noticeTypesForIndex) {
     this.noticeTypesForIndex = noticeTypesForIndex;
   }
 
+  /**
+   * Return the set of notice subtypes for which we expect to have a notice type definition.
+   * This is hardcoded here because there is no other place that has this information.
+   */
+  public Set<String> getExpectedNoticeSubtypes() {
+    return Stream.of("1", "2", "3", "4", "5", "6", "7", "8", "9",
```

Comment:
This is OK for now, maybe later we could have this in a config file?

Reply:
A config file would need to be included as a resource in the JAR, to provide correct default values.
I did try to add this list in the drools file, but it was too cumbersome.
To be seen when we do need to change this.